repos/system_upgrade/el7toel8/actors/addupgradebootentry/libraries/library.py

```diff
@@ -30,3 +30,21 @@ def get_boot_file_paths():
                                       details={'details': 'Did not receive a message about the leapp-provided'
                                                           'kernel and initramfs'})
     return boot_content.kernel_path, boot_content.initram_path
+
+
+def fix_grub_config_error():
+    lines = []
+    conf_file = '/etc/default/grub'
+    with open(conf_file, 'r') as f:
+        lines = f.readlines()
+        for i, line in enumerate(lines):
+            if not line.startswith('GRUB_CMDLINE_LINUX'):
```

Comment:
This won't work in all cases. Per https://www.gnu.org/software/grub/manual/grub/html_node/Simple-configuration.html the content of `/etc/default/grub` "must be valid POSIX shell input" which is to be possible to source by a shell script.
Thus, the valid content of the file could be also just a one-liner:
```
GRUB_DEFAULT=saved GRUB_DISABLE_SUBMENU=true GRUB_TERMINAL_OUTPUT="console" GRUB_CMDLINE_LINUX="vconsole.font=latarcyrheb-sun16 $([ -x /usr/sbin/rhcrashkernel-param ] && /usr/sbin/rhcrashkernel-param || :) vconsole.keymap=us rhgb quiet" GRUB_DISABLE_RECOVERY="true"
```
Source: https://docs.fedoraproject.org/en-US/Fedora/23/html/Multiboot_Guide/GRUB-configuration.html